.github/pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+### Summary
+[PLEASE REMOVE] Describe the goal of this PR and any add any relevant context.
+
+[PLEASE REMOVE] If this PR closes an issue, please add a `Fixes #<issue-id>` line.
+
+[PLEASE REMOVE] If this PR introduces a fix or feature that should be the upcoming release notes, please add the following "Release notes: <area>" label.
+
+### Test plan
```

Comment:
The PR-to-diff importer looks for a section beginning with `Test Plan:`. I don't know if it will detect and respect this `###` formatting.

Reply:
It works with ###, please check the PR description

Reply:
Nice! Thanks for checking that; great to know.